changelog/unreleased/kong/proxy-cache-fix-age-header.yml

```diff
@@ -0,0 +1,4 @@
+message: |
+  **proxy-cache**: Fixed a bug where the Age header was not being updated correctly when serving
```

Comment:
Is the changelog entry incomplete?

Reply:
No, I don't think it's incomplete. Why would it?

Reply:
Ah, I can see where the confusion comes from. Updated the changelog